docs/README

```diff
@@ -1,13 +1,28 @@
 Root of sphinx-build files here for the Longturn Documentation Project
     https://github.com/orgs/longturn/projects/1
 
+LINUX::
+
 To create a local test build run this command from this same directory
 
     cd build
+    (once) pip install virtualenv
```

Comment:
I can already see people typing `(once)` in the terminal.
```suggestion
    pip install virtualenv  # Only the first time
```
And I wouldn't recommend installing anything via `pip` if you can install it with `apt`...